documentation/asciidoc/topics/ref_xsite_crd.adoc

```diff
@@ -40,6 +40,8 @@ ifdef::productized[]
 <5> Specifies a backup location that matches `.spec.service.sites.local.name`.
 <6> Specifies the URL of the {openshiftshort} API for the backup location.
 <7> Specifies the secret that contains the service account token for the backup site.
-<8> Logs error messages for the JGroups TCP protocol.
-<9> Logs fatal messages for the JGroups RELAY2 protocol.
+<8> Directly specifies the `host` routing to another {brandname} backup instance.
```

Comment:
> can we show that <6><7> and <8><9> aren't in sequence but two different options?

Sure! I'm not very familiar with ascidoc, so formatting suggestions are welcome.

Reply:
I would use <6a><7a> and <6b><7b> if asciidoc doesn't complain

Reply:
@rigazilla unfortunately that' wouldn't work. asciidoc would treat <6a> as text rather than a callout. 

The more I think about it, the more I feel like we need two new user stories, which means two new assemblies. One user story for the k8s api approach, which means the operator looks up the connections and controls them. Another user story for the direct xsite connection. That way we can clearly show they are two distinct options and explain the pros and cons.

Reply:
I agree. A separate user story for direct xsite would allow us to emphasize that this approach can be useful for communicating with non-k8 clusters as well.